3rdparty/evercrypt

```diff
@@ -0,0 +1 @@
+../evercrypt-msr/evercrypt/
```

Comment:
Is it bad practice to have a submodule live in `3rdpary/`?

Reply:
No but we can't submodule just the directory, so it would be quite messy.